rpc/node.go

```diff
@@ -90,7 +97,10 @@ func (node *Node) Info(arguments *InfoArguments, reply *InfoReply) error {
 	incoming, outgoing := peer.GetCounts()
 	reply.Chain = mode.ChainName()
 	reply.Mode = mode.String()
-	reply.Blocks = blockheader.Height()
+	reply.Blocks = blockInfo{
```

Comment:
> incompatible API change so need to leave to version 0.11.x

What is the purpose?

>do you think we should keep blocks as a number and add separate hash field to keep 0.10.x compatibility so the webinfo systems don not have to be changed

I think we should keep it in `blockInfo` since we maybe have block related stuff need to expose here later.